project/src/components/main/cards/cards.tsx

```diff
@@ -0,0 +1,39 @@
+import Card from '../card/card';
+import { Offers } from '../../../../src/types/types';
+
+type cardsProps = {
+  offers: Offers,
+}
+
+function Cards( { offers }: cardsProps): JSX.Element {
+  return (
+    <div className="cities__places-list places__list tabs__content">
+      {offers.map(({
+        id,
+        title,
+        price,
+        type,
+        rating,
+        isPremium,
+        isFavorite,
+        previewImage }) =>
+        (
+          <Card
+            key={id}
+            type={type}
+            title={title}
+            price={price}
+            rating={rating}
+            isPremium={isPremium}
+            isFavorite={isFavorite}
+            previewImage={previewImage}
+            id={id}
+          />),
+      )}
```

Comment:
ох, тебе самому не страшно на такое смотреть?) 
```suggestion
      {offers.map(({id, ...rest}) => <Card id={id} key={id} {...rest} />)}
```